python/cudf_kafka/cudf_kafka/_lib/kafka.pxd

```diff
@@ -34,7 +34,7 @@ cdef extern from "kafka_consumer.hpp" \
         int64_t get_committed_offset(string topic,
                                      int32_t partition) except +
 
-        map[string, vector[int32_t]] list_topics(string topic) except +
+        map[string, vector[int32_t]] list_topics(string topic) except +*
```

Comment:
Since this is a binding to an external C++ function, it can only possibly raise a C++ exception. Thus, I think we should mark it `except +`.

Reply:
`except +` ensures that C++ exceptions get translated to Python exceptions.